packages/xs-extra/xapi-rrdd.master/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "1"
+maintainer: "john.else@eu.citrix.com"
+build: [
+  [make]
+  [make "install" "BINDIR=%{bin}%" "SBINDIR=%{bin}%" "LIBEXECDIR=%{bin}%" "SCRIPTSDIR=%{bin}%" "ETCDIR=%{prefix}%/etc"]
+]
+build-test: [
+  [make "test"]
+]
+remove: [make "uninstall" "BINDIR=%{bin}%" "SBINDIR=%{bin}%" "LIBEXECDIR=%{bin}%" "SCRIPTSDIR=%{bin}%" "ETCDIR=%{prefix}%/etc"]
+depends: [
+  "obuild" {>= "0.1.1"}
+  "ocamlfind"
+  "xapi-backtrace"
+  "xapi-idl" {>= "0.9.21"}
+  "xapi-libs-transitional" {>= "0.9.7"}
+  "xapi-inventory"
+  "xapi-stdext" {>= "0.13.0"}
+  "xapi-forkexecd"
+  "xenctrl" {>= "0.9.28"}
+  "xenstore_transport"
+  "xapi-xenops"
+  "io-page"
+  "xen-gnt"
+  "xapi-rrd-transport"
+  "oclock"
+  "ounit"
+  "bisect_ppx"
+  "systemd"
+]
+ocaml-version: [>= "4.01.0"]
```

Comment:
This should be 4.02.3 givent that we use `bisect_ppx`. Probably we can drop it altogether